### ISSUE TITLE
.populate() creating empty associations

### ISSUE THREAD

**Issue by randallmeeker** [balderdashy/sails#1833]
Account.find().populate('lastAssignment')

When I use populate to get the last assignment on the model below, it will also will also attach an empty array of assignments. If I populate assignments ```Account.find().populate('assignments')``` all is right with the world. 

```
module.exports = {
  migrate: 'safe',
  tableName: 'accounts',
  autoPK: false,
  attributes: {
		id: {type: 'INTEGER', primaryKey: true},
		companyId: 'INTEGER',
		name: 'STRING',		

		// Associations

		company: {
			model: 'company',
			columnName: 'companyId'
		},

		assignments: {
			collection: 'assignment',
			via: 'account'
		},

		lastAssignment:{
			model: 'assignment',
			columnName: 'lastAssignmentId'
		}

	}
};
```

**Comment by mikermcneil** [comment]
Thanks for posting, @randallmeeker.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
